### ISSUE TITLE
Scipy Misses Most Functions

### ISSUE THREAD

**Issue by AeneasZhu** [microsoft/python-language-server#1824]
![image](https://user-images.githubusercontent.com/44701446/70107444-79e71100-1681-11ea-8006-cf7eeb860ae2.png)

![image](https://user-images.githubusercontent.com/44701446/70107464-84a1a600-1681-11ea-9291-5e64f38ff405.png)

Is this 'bug' same as numpy's missing functions?

**Comment by MikhailArkhipov** [comment]
```python
class norm_gen(rv_continuous): ...
...
norm = norm_gen(name='norm')
```
appears to be a regular class. In 0.5+ I see

![image](https://user-images.githubusercontent.com/12820357/70929718-91f75280-1fe8-11ea-86a1-600ce6002892.png)

Could you try
```json
"python.analysis.downloadChannel": "beta"
```
The setting will be grayed out as it is primary for development. Thanks.

**Comment by AeneasZhu** [comment]
> ```python
> class norm_gen(rv_continuous): ...
> ...
> norm = norm_gen(name='norm')
> ```
> 
> appears to be a regular class. In 0.5+ I see
> 
> ![image](https://user-images.githubusercontent.com/12820357/70929718-91f75280-1fe8-11ea-86a1-600ce6002892.png)
> 
> Could you try
> 
> ```json
> "python.analysis.downloadChannel": "beta"
> ```
> 
> The setting will be grayed out as it is primary for development. Thanks.


By the `"python.analysis.downloadChannel": "beta"` doesn't show in the setting json

**Comment by MikhailArkhipov** [comment]
Yes. As I said, the setting will be grayed out as it is primary for development. It is not provided in completions.
